src/System.Runtime/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -2,3 +2,6 @@ Compat issues with assembly System.Runtime:
 TypeCannotChangeClassification : Type 'System.Decimal' is marked as readonly in the contract so it must also be marked readonly in the implementation.
 TypeCannotChangeClassification : Type 'System.Double' is marked as readonly in the contract so it must also be marked readonly in the implementation.
 TypeCannotChangeClassification : Type 'System.Single' is marked as readonly in the contract so it must also be marked readonly in the implementation.
+TypeCannotChangeClassification : Type 'System.DateTimeOffset' is marked as readonly in the contract so it must also be marked readonly in the implementation.
+TypeCannotChangeClassification : Type 'System.TimeSpan' is marked as readonly in the contract so it must also be marked readonly in the implementation.
```

Comment:
src/System.Runtime.InteropServices/src/ApiCompatBaseline.uapaot.txt and src/System.Threading/src/ApiCompatBaseline.uapaot.txt will need similar change.